system-contracts/test/MsgValueSimulator.spec.ts

```diff
@@ -0,0 +1,194 @@
+import type { L2EthToken, MockContract, MsgValueSimulator } from "../typechain";
+import { L2EthTokenFactory, MsgValueSimulatorFactory } from "../typechain";
+import { deployContract, deployContractOnAddress, getWallets, loadZasmBytecode, setCode } from "./shared/utils";
+import {
+  TEST_BOOTLOADER_FORMAL_ADDRESS,
+  TEST_ETH_TOKEN_SYSTEM_CONTRACT_ADDRESS,
+  TEST_MSG_VALUE_SYSTEM_CONTRACT_ADDRESS,
+} from "./shared/constants";
+import { prepareEnvironment } from "./shared/mocks";
+import { expect } from "chai";
+import { EXTRA_ABI_CALLER_ADDRESS, encodeExtraAbiCallerCalldata } from "./shared/extraAbiCaller";
+import { BigNumber } from "ethers";
+import { Contract } from "zksync-web3";
+import * as hardhat from "hardhat";
+
+describe("MsgValueSimulator tests", () => {
+  let messageValueSimulator: MsgValueSimulator;
+  let extraAbiCaller: Contract;
+  let mockContract: MockContract;
+  let l2EthToken: L2EthToken;
+  const wallet = getWallets()[0];
+  const msgSender = "0x000000000000000000000000000000000000beef";
```

Comment:
Is it equals to the `EXTRA_ABI_CALLER_ADDRESS`?

Reply:
Right, removed it.

Reply:
Hmm, have you commited this? I still see this constant